source/slang/slang.cpp

```diff
@@ -912,6 +912,9 @@ RefPtr<ModuleDecl> CompileRequest::loadModule(
     if( errorCountAfter != errorCountBefore )
     {
         mSink.diagnose(srcLoc, Diagnostics::errorInImportedModule);
+    }
+    if (errorCountAfter)
+    {
```

Comment:
This seems weird, because it means that if there were *any* errors (even if they weren't in the thing we `import`ed, we will bail out from `import`ing). I know our behavior around `import` and errors has been flaky in various ways for some time, so I'm fine with anything that improves robustness, but this seems like something that needs a deep dive and a real fix (based on a clear set of requirements that ensure the desired user experience) sooner or later.

Reply:
Yeah, but I think that there are many "real" causes of such kind of crash, so it is unlikely that we are able to catch and fix them all. For robustness this seemed to be necessary.